Exercises/chapter3/e2-fizzbuzz.js

```diff
@@ -0,0 +1,12 @@
+for (let i = 0; i < 100; i++) {
+  let output = "";
+  if (i % 15 == 0) {
```

Comment:
In JS you should always use `===`. You can google it to find plenty of blog posts on why